### PR TITLE
The JVM args started with "java" but it should just start at -Xms

### PR DESCRIPTION
**Pull request by JWGardiner**

### CODE REVIEW

**Comment by lrmtheboss** [comment]
When typing it into a terminal you need java at the start so it knows to use java as the program, on game management such as pterodactyl you also still need it

**Comment by JWGardiner** [comment]
Ah okay, I made some more changes to clarify that

File: JavaArgumentsOptimization.md [hunk 1]
```diff
@@ -22,8 +22,10 @@ These flags are made by Aikar and you can find the explanation for the flags [he
 
 Use these flags exactly, only changing Xmx and Xms. These flags work and scale accordingly to any size of memory.
 
+If you're using the vanilla launcher, remove java from the start. Some other laumchers require java at the start
```
Comment (lrmtheboss) [inline JavaArgumentsOptimization.md:25]: `laumchers` please fix the spelling

**Review by lrmtheboss — CHANGES_REQUESTED** [review]
`laumchers` please fix the spelling
It may be better to say some launchers require "java" at the start others do not
On that same note, this does not only apply to launchers, thus a more generic term would be best.

File: JavaArgumentsOptimization.md (hunk 1 above)
Comment (JWGardiner) [reply JavaArgumentsOptimization.md]: alright, I have made it more general, so it can apply to anything

**Review by lrmtheboss — APPROVED** [review]

**Pull request merged**